packages/edge-bundler/node/bridge.ts

```diff
@@ -16,7 +16,7 @@ const DENO_VERSION_FILE = 'version.txt'
 // When updating DENO_VERSION_RANGE, ensure that the deno version
 // on the netlify/buildbot build image satisfies this range!
 // https://github.com/netlify/buildbot/blob/f9c03c9dcb091d6570e9d0778381560d469e78ad/build-image/noble/Dockerfile#L410
-const DENO_VERSION_RANGE = '1.39.0 - 1.46.3'
+const DENO_VERSION_RANGE = '1.39.0 - 2.2.4'
```

Comment:
Do note, that if system doesn't have deno installed, or installed version doesn't match the range, we will continue to download `1.39.0` for our use due to way this is implemented:
https://github.com/netlify/build/blob/17cdb7c56374a25d596ceec7fd62ab810b04f974/packages/edge-bundler/node/downloader.ts#L90-L108

And fact that actually latest version of Deno today is 2.2.6 (and will continue to be bumped), so that latest version is not in our range and the function falling back to min version.

I think this might be desirable to be the case at least temporarily to possibly migrate with slower pace to catch things we didn't catch here, but I just want to at least flag it here